devel/swiftlint/Portfile

```
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcodeversion 1.0
+
+github.setup        realm swiftlint 0.52.4
+
+name                swiftlint
+version             0.52.4
+categories          devel
+platforms           darwin
+license             MIT
+maintainers         gmail.com:christos.koninis
```

A port should also be marked `openmaintainer` unless there is a very specific reason why you and only you should update or make any changes to the port:

```suggestion
maintainers         gmail.com:christos.koninis \
                    openmaintainer
```

---

and please add your GitHub handle

---

@reneeotten Nice I missed that about the maintainers syntax. I added my Github handle